docs/tutorials/4_train_airl.ipynb

```diff
@@ -127,7 +128,7 @@
     "    reward_net=reward_net,\n",
     ")\n",
     "\n",
-    "env.reset(seed=SEED)\n",
+    "env.reset()\n",
```

Comment:
Why are we removing the seed here? I'd be nervous this might make the test flaky.

Reply:
Because the venv created from [`make_vec_env`](https://github.com/HumanCompatibleAI/imitation/blob/master/src/imitation/util/util.py#L80) doesn't accept seed.

Reply:
I updated the naming from `env` to `venv` to make things clearer. Also, I added `venv.seed` before `evaluate_policy` to make sure the experimentation can be the same. I ran the notebook twice and they both shown:

```
Rewards before training: 102.6 +/- 24.11514047232568 
Rewards after training: 319.54 +/- 159.8500184548003
```

Reply:
Oh, I see, that change makes sense. Given we were accidentally not testing any of the notebooks before this the bug must have managed to slip through CI (it's certainly not Windows specific).